docs/source/config.json

```diff
@@ -8,7 +8,7 @@
     "Get Started": "/",
     "Migrating to v4": "/migration/4.0",
     "Evolution policy": "/essentials/evolution",
-    "Kdoc": "https://www.apollographql.com/docs/kotlin/kdoc/",
+    "Kdoc": "https://apollographql.github.io/apollo-kotlin/kdoc/",
```

Comment:
External links have a small icon next to them which I think is expected here since the KDoc is quite separate (there's no way to navigate to other sections without clicking "back"). Also unifies the KDoc link with Apollo Galaxy projects.